integration-tests/tests/test_wallet_rpc.py

```diff
@@ -39,14 +39,45 @@ def test_tx_(self, method, params):
 
     def test_create_multi_transaction(self):
         response = self.wallet_create_multi_transaction()
+        self.verify_is_valid_json_rpc_response(response)
         
         # how to create schema:
         # from schema_builder import CustomSchemaBuilder
         # CustomSchemaBuilder(method).create_schema(response.json())
         
-        with open(f"{option.base_dir}/schemas/wallet_createMultiTransaction", "r") as schema:
+        with open(f"{option.base_dir}/schemas/wallet_createMultiTransaction/transferTx_positive", "r") as schema:
             jsonschema.validate(instance=response.json(), schema=json.load(schema))
 
+    @pytest.mark.parametrize(
+        "method, changed_values, expected_error_code, expected_error_text",
+        [
+            (
+                    "transferTx_value_not_enough_balance",
+                    {'value': '0x21e438ea8139cd35004'}, -32000, "Insufficient funds for gas",
+            ),
+            (
+                    "transferTx_from_from_invalid_string",
+                    {'from': 'some_invalid_address'}, -32602, "cannot unmarshal hex string without 0x prefix",
+            ),
+        ],
+    )
+    def test_create_multi_transaction_validation(self, method,
+                                                 changed_values,
+                                                 expected_error_code, expected_error_text):
+        response = self.wallet_create_multi_transaction(**changed_values)
+        self.verify_is_json_rpc_error(response)
+        actual_error_code, actual_error_text = response.json()['error']['code'], response.json()['error']['message']
+        if expected_error_code != actual_error_code:
```

Comment:
I would prefer to have native asserts, e.g.
`assert  actual_error_code != expected_error_code, f"got code: {actual_error_code} instead of expected: {expected_error_code}"`

as result error message will be 
```
AssertionError: got code: -32000 instead of expected: 20000
```

also saves some space :)

Reply:
done